modules/sharedaddy/sharing-sources.php

```diff
@@ -1537,7 +1552,9 @@ public function display_footer() {
 				var s = document.createElement("script");
 				s.type = "text/javascript";
 				s.async = true;
-				<?php if ( $jetpack_pinit_over ) { echo "s.setAttribute('data-pin-hover', true);";} ?>
+                <?php if ( $jetpack_pinit_over ) { 
```

Comment:
Tabs instead of spaces.

Also, I believe the `echo` should be indented once more than the conditional. 

Lastly, I prefer the following style, although yours works just as well.

```php
<?php if ( $jetpack_pinit_over ):  
	echo "s.setAttribute('data-pin-hover', true);";
endif; ?>
```